### ISSUE TITLE
MD033 false positive with email

### ISSUE THREAD

**Issue by regseb** [DavidAnson/markdownlint#183]
The [email autolink](https://github.github.com/gfm/#email-autolink) elements are considered as HTML tags by the [MD033](https://github.com/DavidAnson/markdownlint/blob/master/lib/md033.js) rule.

```markdown
My email : <my@email.com>
```

**Comment by DavidAnson** [comment]
Apologies! This seems likely to be the same issue as #182, I will try to get a fix out this evening.
